kusoCode.rb

```diff
@@ -0,0 +1,5 @@
+def badName
```

Comment:
Naming/FileName: The name of this source file (`kusoCode.rb`) should use snake_case.